breeze/breeze.d.ts

```diff
@@ -1,13 +1,15 @@
 // Type definitions for Breeze 1.4
 // Project: http://www.breezejs.com/
-// Definitions by: Boris Yankov <https://github.com/borisyankov/>, IdeaBlade <https://github.com/IdeaBlade/Breeze/>
+// Definitions by: Boris Yankov <https://github.com/borisyankov/>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 // Updated Jan 14 2011 - Jay Traband ( www.ideablade.com).
-// Updated March 27 2013 - John Lantz (www.ideablade.com).
```

Comment:
John Lantz is no longer at ideablade, so I didn't want anyone to come looking for him.